src/Management/src/Tasks/Steeltoe.Management.Tasks.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net8.0</TargetFrameworks>
-    <Description>Extensions for running tasks embedded in your ASP.NET Core application. Ideal for cf run-task in Cloud Foundry.</Description>
-    <PackageTags>tasks;management;monitoring;aspnetcore;Spring Cloud;cf run-task</PackageTags>
+    <Description>Extensions for running tasks embedded in your .NET application. Ideal for cf run-task in Cloud Foundry.</Description>
+    <PackageTags>tasks;management;Spring Cloud;cf run-task</PackageTags>
```

Comment:
Aren't spaces and semicolons both considered separators by NuGet? If so, "cf run-task" is probably unintended.

Reply:
It would be clearer to always use a semicolon as separator and never spaces. It's way too easy to interpret wrong, because reading an element that contains a mix of semicolons and spaces suggests that the spaces are preserved, which is not the case.

Reply:
ok, spaces are gone